code/sorting/src/postman's sort/postman's_sort.c

```diff
@@ -0,0 +1,102 @@
+// C Program to Implement Postman Sort Algorithm
+
+#include <stdio.h>
+
+int main() 
+{
+    int t1, t2, k, t, n = 1;
```

Comment:
Except for the loop variables, declare all of these variables closest to where they are used.